kong/plugins/session/session.lua

```diff
@@ -112,7 +112,7 @@ function _M.logout(conf)
         local logout_post_arg = conf.logout_post_arg
         if logout_post_arg then
           local post_args = kong.request.get_body()
```

Comment:
I don't know why the tests were passing before, but this will fail with a not-so-clear error when the request is missing "Content-Type" header.

Reply:
@locao fixing it, it is the tests that patch `kong.request.*`. But sure, interesting that tests were passing before...

Reply:
@locao seems like it passes now.